doc/BUILDSERVER.md

```diff
@@ -0,0 +1,98 @@
+# Buildserver
+
+## Introduction
+Most of the build jobs on the jenkins server that handles automated testing of
+contributions have been replaced by a single
+[job](https://github.com/ElektraInitiative/libelektra/).
```

Comment:
This does not really introduce what the buildserver does.

Is the link as wanted? (I would expect links to the build server in the intro of the buildserver)

Reply:
I will take a step back and describe the whole setup not just the new one.

Reply:
Thank you!